R/writeComps.R

```diff
@@ -28,14 +28,17 @@
 #'   later. Note that \code{maxAge} is only used if \code{abins = NULL}, otherwise
 #'   fish are binned according to user specified bins irregardless of \code{maxAge}.
 #'   
+#' @param month Month for all observations. Defaults to 7. If input has multiple 
+#' seasons, this must be a vector of equal length to the number of seasons.
+#' 
```

Comment:
Can you clarify this to me? If 'season' is included in the defaults list passed to the getComps function, is this when you would need to provide a vector of equal length to the number of season. If so, I think we should add this piece of information.

Reply:
The code I'm using, based on your UW class example, has a "season" column in the output from 
`length_comps <- getComps(Pdata = ...)`
where all the season values = 1.
However, if you first run `Pdata_seas <- getSeason(Pdata, season_type = 1, yearUp = c(11, 12))` then the season values are 1 or 2. I'm sure there are less confusing ways to describe this, but as implemented, the `month = 7` default should always result in taking the first element of that value treated as a vector: `c(7)[1] = 7`. And if you have 2 seasons, then an input like `month = c(1,7)` will result in `c(1,7)[season] = 1 or 7`.

Reply:
note: working example with and without seasons (up through length comps, not yet age comps) is available here: https://github.com/pfmc-assessments/petrale/blob/main/R/process_pacfin_bds.R